src/CSET/recipes/transect.yaml

```diff
@@ -26,9 +26,14 @@ parallel:
 
 collate:
   - operator: read.read_cube
+    constraint:
+        operator: constraints.generate_var_constraint
+        varname: $CS_VARS
+
     filename_pattern: intermediate/*
 
   - operator: write.write_cube_to_nc
+    filename: transect_$CS_VARS.nc
```

Comment:
The files will be written out to separate folders now the title is different, so this isn't needed. On the other hand it may be clearer to keep it, so I'll leave it up to you.